### ISSUE TITLE
GGML_ASSERT(logits != nullptr)

### ISSUE THREAD

**Issue by elibroftw** [NexaAI/nexa-sdk#1269]
**Describe the bug**
When trying to infer the model `DavidAU/Qwen3.6-27B-Fable-Fusion-711-Uncensored-Heretic-NM-DAU-NEO-MAX-MTP-GGUF`, it crashes

**To Reproduce**

```
PS C:\Users\maste> geniex infer DavidAU/Qwen3.6-27B-Fable-Fusion-711-Uncensored-Heretic-NM-DAU-NEO-MAX-MTP-GGUF
Model is not currently cached, downloading...
downloading 100% |█████████████████████████████████████████████████████████████████████| (19/19 GB, 41 MB/s)

   Detected model type: vlm
✔  Download success
> TEST
C:/a/GenieX/GenieX/third-party/llama.cpp/common/sampling.cpp:154: GGML_ASSERT(logits != nullptr) failed
```

**Expected behavior**

WORKS

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 11 ARM

**Additional context**

**Comment by mansiverma897993** [comment]
@mengshengwu I have raised #1273 let me know further changes to fix this !!
